modules/cert-manager-configure-cloud-credentials-aws-sts.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-authenticate-aws.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-configure-cloud-credentials-aws-sts"]
+= Configuring the {cert-manager-operator} to use prepared cloud credentials
+
+You can configure the {cert-manager-operator} to use prepared cloud credentials on AWS Security Token Service cluster by using the `ccoctl` binary.
+
+.Prerequisites
+
+* You must extract and prepare the `ccoctl` binary.
+
+.Procedure
+
+. The {cert-manager-operator} creates the `CredentialsRequest` object in the `openshift-cloud-credential-operator` namespace. Get the role by running the following command:
++
+[source,terminal]
+----
+$ oc get credentialsrequest -n openshift-cloud-credential-operator -o yaml
+----
++
+.Example output
+[source,terminal]
+----
+arn:aws:iam::<AWS-account-id>:role/abcd-cert-manager
+----
+
+. Annotate the service account by running the following command:
++
+[source,terminal]
+----
+$ oc annotate sa/cert-manager -n cert-manager eks.amazonaws.com/role-arn="arn:aws:iam::<AWS-account-id>:role/abcd-cert-manager"
```

Comment:
Let's add the three other annotations in more steps as well.
Please refer to step 4 from https://github.com/openshift/cert-manager-operator/blob/master/docs/cloud_credentials.md#aws (for the other 3 annotations prescribed in https://github.com/openshift/aws-pod-identity-webhook#eks-walkthrough)